imports/plugins/included/product-variant/server/methods/populateTaxCodes.js

```diff
@@ -0,0 +1,64 @@
+import { Meteor } from "meteor/meteor";
+import { TaxCodes } from "/imports/plugins/core/taxes/lib/collections";
+
+const taxCodes = {};
+
+/*
+ * taxes/insertTaxCodes
+ * @summary populate TaxCodes collection
+ * @param {String} shopID - current shop's id
+ * @param {Object} code - tax code object to insert into TaxCodes collection
+ * @param {String} providerName - tax code provider
+ * @return {} undefined
+ */
+taxCodes.populateTaxCodes = function (shopId, code, providerName) {
+  check(shopId, String);
+  check(code, Object);
+  check(providerName, String);
+
+  try {
+    TaxCodes.insert({
+      id: code.id,
+      shopId: shopId,
+      taxCode: code.taxCode,
+      taxCodeProvider: providerName,
+      ssuta: code.isSSTCertified,
+      label: code.description,
+      parent: code.parentTaxCode
+    });
+  } catch (e) {
+    throw e;
```

Comment:
Let's try to throw a `Meteor.Error` with a better description of where it happened and what happened.